modules/jupyter/jupyter_config/config-selfauth.yaml

```diff
@@ -30,6 +30,11 @@ hub:
       authenticator_class: ${authenticator_class}
     GCPIAPAuthenticator: 
       header_name: 'X-Goog-IAP-JWT-Assertion'
+      project_id: '${project_id}'
+      namespace: '${namespace}'
+      service_name: '${service_name}'
+      project_number: '${project_number}'
+      # support legacy image
       expected_audience: '/projects/${project_number}/global/backendServices/${service_id}'
```

Comment:
this will be generated automatically, we dont need to pass it?

Reply:
support legacy, because I can not modify the repo the same time, after this change merged, I will update repo and remove this line